build.sbt

```diff
@@ -251,11 +251,14 @@ lazy val httpTests = project("akka-http-tests")
 
 lazy val httpJmhBench = project("akka-http-bench-jmh")
   .settings(commonSettings)
+  .settings(
+    whitesourceIgnore := true
+  )
   .dependsOn(http, http2Support % "compile->compile,test")
   .addAkkaModuleDependency("akka-stream")
   .enablePlugins(JmhPlugin)
   .enablePlugins(NoPublish).disablePlugins(BintrayPlugin) // don't release benchs
```

Comment:
We could also just add the setting to `NoPublish`, WDYT?

Reply:
that makes sense to me :+1:

Reply:
Done